apps/rush-lib/package.json

```diff
@@ -18,6 +18,8 @@
   },
   "license": "MIT",
   "dependencies": {
+    "@azure/identity": "~1.2.0",
+    "@azure/storage-blob": "~12.3.0",
```

Comment:
This brings in 2,844 `node_modules` files.  At some point we need to figure out a way for Rush to install dependencies on demand.

Reply:
Yeah agreed.

Reply:
The [AWS SDK](https://www.npmjs.com/package/aws-sdk) is 57MB, so we need to figure this out pretty quickly.

Reply:
Can you open a GitHub issue where we can discuss the design?  I'm not sure at all what is the best solution.

Reply:
https://github.com/microsoft/rushstack/issues/2408